src/main/java/com/puppycrawl/tools/checkstyle/Checker.java

```diff
@@ -481,24 +469,23 @@ protected void setupChild(Configuration childConf)
         }
         catch (final CheckstyleException ex) {
             throw new CheckstyleException(
-                    getLocalizedMessage("Checker.setupChildModule", name, ex.getMessage()), ex);
+                getLocalizedMessage("Checker.setupChildModule", name, ex.getMessage()), ex);
         }
+
```

Comment:
might be an format issue:

<img width="1750" alt="image" src="https://github.com/user-attachments/assets/5f8b9b69-0f1c-4f24-bdf2-ac3cbd124505" />